conf/flight_plans/lagonisi_beach_survey.xml

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE flight_plan SYSTEM "flight_plan.dtd">
+
+<flight_plan alt="200." ground_alt="42" lat0="37.7776225" lon0="23.9184623" max_dist_from_home="4000" name="lagonisi_beach_survey" qfu="180" security_height="100.">
+  <header>
+#include "subsystems/ins/ins_alt_float.h"
+#include "subsystems/datalink/datalink.h"
+#include "generated/airframe.h"
+#if defined(COMMAND_FLAPS)
+#define Flaps(x) ({ (*ap_state).commands[COMMAND_FLAPS] = (int16_t)((float)MAX_PPRZ * (float)x); 0; })
+#else
+#define Flaps(x)    {0;}
+#endif
+#define RcChannel(x) ((*fbw_state).channels[RADIO_ ## x])
+</header>
```

Comment:
FIXED by removing the FLAPS setting.
When i use the function "imcu_set_command(COMMAND_FLAPS, 0)" inside the flight plan like this <call fun="imcu_set_command(COMMAND_FLAPS, MAX_PPRZ/2)"/>
I get this error
/home/hendrix/paparazzi/var/aircrafts/Easyglider_matek/ap/generated/flight_plan.h:202:13: error: invalid use of void expression  if (! (imcu_set_command(COMMAND_FLAPS, 0))) {

The imcu_get_radio(RADIO_X) on the other hand works fine.

Reply:
you should not change this function, but use `call_once` instead of `call` in the flight plan (http://wiki.paparazziuav.org/wiki/Flight_Plans#Call)

Reply:
OK i will fix everything.

After much testing i decided to remove some excess code so do you want me to delete this pull request

and create a fresh one with the final files?

I don't know but because of my perfectionism i tend to mess things up...

Chris